test/engine.test.js

```diff
@@ -604,12 +604,12 @@ describe('Engine', () => {
         `)
             throw new Error('should not be reached')
         } catch (err) {
-            expect(err.message).to.includes('[string "..."]:3: function a threw error')
+            expect(err.message).to.includes('[string "?"]:3: function a threw error')
             expect(err.message).to.includes('stack traceback:')
-            expect(err.message).to.includes(`[string "..."]:3: in upvalue 'a'`)
-            expect(err.message).to.includes(`[string "..."]:5: in upvalue 'b'`)
-            expect(err.message).to.includes(`[string "..."]:6: in local 'c'`)
-            expect(err.message).to.includes(`[string "..."]:7: in main chunk`)
+            expect(err.message).to.includes(`[string "?"]:3: in upvalue 'a'`)
+            expect(err.message).to.includes(`[string "?"]:5: in upvalue 'b'`)
+            expect(err.message).to.includes(`[string "?"]:6: in local 'c'`)
+            expect(err.message).to.includes(`[string "?"]:7: in main chunk`)
```

Comment:
How come these switch to question marks?

Reply:
That was a optimization on loadString to send the name as null, by default. But I'm going to rollback this because the default on several parts of the internal lua code is sending the code itself.